cmd/certs/main.go

```diff
@@ -40,31 +44,46 @@ import (
 const (
 	stopWaitTime = 5 * time.Second
 
-	defLogLevel      = "error"
-	defDBHost        = "localhost"
-	defDBPort        = "5432"
-	defDBUser        = "mainflux"
-	defDBPass        = "mainflux"
-	defDB            = "certs"
-	defDBSSLMode     = "disable"
-	defDBSSLCert     = ""
-	defDBSSLKey      = ""
-	defDBSSLRootCert = ""
-	defClientTLS     = "false"
-	defCACerts       = ""
-	defPort          = "8204"
-	defServerCert    = ""
-	defServerKey     = ""
-	defCertsURL      = "http://localhost"
-	defThingsURL     = "http://things:8182"
-	defJaegerURL     = ""
-	defAuthURL       = "localhost:8181"
-	defAuthTimeout   = "1s"
-
-	defSignCAPath     = "ca.crt"
-	defSignCAKeyPath  = "ca.key"
-	defSignHoursValid = "2048h"
-	defSignRSABits    = ""
+	defLogLevel              = "error"
+	defDBHost                = "localhost"
+	defDBPort                = "5432"
+	defDBUser                = "mainflux"
+	defDBPass                = "mainflux"
+	defDB                    = "certs"
+	defDBSSLMode             = "disable"
+	defDBSSLCert             = ""
+	defDBSSLKey              = ""
+	defDBSSLRootCert         = ""
+	defClientTLS             = "false"
+	defCACerts               = ""
+	defPort                  = "8204"
+	defServerCert            = ""
+	defServerKey             = ""
+	defCertsURL              = "http://localhost"
+	defThingsURL             = "http://things:8182"
+	defJaegerURL             = ""
+	defAuthURL               = "localhost:8181"
+	defAuthTimeout           = "1s"
+	defSignCAPath            = "ca.crt"
+	defSignCAKeyPath         = "ca.key"
+	defSignHoursValid        = "2048h"
+	defSignRSABits           = "2048"
+	defCertAutoRenew         = true
+	defCertAutoRenewUpdateBS = true
+	defStopSvcOnRenewErr     = true
+
+	defBootstrapURL = "http://bootstrap:8202/things/configs/certs"
+	defMFUser       = "test@email.com"
+	defMFPass       = "123"
+
+	defThingsESURL  = "localhost:6379"
+	defThingsESPass = ""
+	defThingsESDB   = "0"
+
+	defESURL          = "localhost:6379"
+	defESPass         = ""
+	defESDB           = "0"
```

Comment:
Looks like these are not used.

Reply:
Still Workin Progress , ES is used to get notification of deleted things

Reply:
removed unsed